.github/workflows/documentation.yml

```diff
@@ -20,7 +20,8 @@ jobs:
       uses: dschep/install-pipenv-action@v1
     - name: Install dependencies
       run: pipenv install --dev    
-    - uses: ammaraskar/sphinx-action@master
+    - name: Build docs      
+      uses: ammaraskar/sphinx-action@master
```

Comment:
@meilisearch/integration-team be careful using `master` tag in github action. If you don't have the choice it's ok, but always prefer alternatives. You can have bad surprise without noticing it with `master` if the maintainer is merging a buggy commit to `master`

Reply:
This is the way it gives in the doc, I will check how to use it differently

Reply:
Yes I saw, that's why I mention to check alternatives (not an emergency, just to warn you)